src/i18n.js

```diff
@@ -12,6 +13,7 @@ const resources = {
   en,
   'zh-CN': zhCn,
   'zh-TW': zhTw,
+  fr
```

Comment:
Just two minor changes needed to make eslint happy and get the Travis build to pass. 
1) move this key above `'zh-CN'`
2) add a trailing comma after `fr`

Reply:
Ok, I will fix this asap (tomorrow at the latest)

Reply:
oh just saw it has already been done! Thanks @mejackreed

Reply:
I just made the minor change as a new commit to this branch. I hope that is ok.

Reply:
@regisrob! It turns out that maintainers can make edits to a PR from your fork. 
https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork 

@mejackreed made the changes so we can ship this with today's release 🎉